docs/en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -11,6 +11,68 @@ The examples below are using certain folder naming conventions (CSS files in `cs
 SilverStripe core modules like `cms` use a different naming convention (CSS and JavaScript files in `client/src/`).
 The `Requirements` class can work with arbitrary file paths.
 
+## Exposing static assets
+
+Before requiring a static assets file in PHP code or in a template, those assets need to be "exposed". This process allows SilverStripe projects and SilverStripe modules to make static asset files available via the web server without directly exposing PHP files.
+
+### Configuring your project "exposed" folders
+
+Exposed assets are made available in your web root in a dedicated "resources" directory. Prior to SilverStripe 4.4, the name of this directory was hardcoded to `resources`. In SilverStripe 4.4 and above, the name of the resources directory can be configured by defining the `extra.resources-dir` key in your `composer.json`. SilverStripe projects created from `silverstripe/installer` 4.4 and above will automatically be configured to use `_resources` as their resource directory.
+
+Each folder that needs to be exposed must be entered under the `extra.expose` key in your `composer.json` file. Module developers should use a path relative to the root of their module (don't include the "vendor/package-developer/package-name" path).
+
+This is a sample `composer.json` configured to expose some assets.
+
+```json
+{
+    "name": "app/myproject",
+    "type": "silverstripe-project",
+    "require": {
+        "silverstripe/recipe-cms": "4.4.x-dev"
+    },
+    "extra": {
+        "resources-dir": "_resources",
+        "expose": [
+            "app/client/dist",
+            "app/images"
+        ]
+    }
+}
+```
+
+Files contained inside the `app/client/dist` and `app/images` will be made publicly available under the `_resources` directory.
+
+SilverStripe projects should not track the "resources" directory in their source control system.
```

Comment:
This is slightly opinionated to be honest - it depends on your deployment process. If you're deploying and running `composer install`, then you shouldn't track it in VCS, but if you're not using composer on the production server you're going to be committing in the vendor and resources folders. I agree that this is true for the majority of use cases though.

Reply:
There's nothing wrong with being opinionated when you are right! :stuck_out_tongue_winking_eye: 

The description will be appropriate for 90% of developers. I'm keen on avoiding going off on a tangent about "if you've got a weird GIT set up, you should do this ..."

If a developer decide to deviate from best practices, they ought to understand why and adjust their set up accordingly.

Reply:
Yeah fair enough!